content/blog/2017/2017-01-20-blueocean-dev-log-jan2.adoc

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Blue Ocean Dev Log: January Week #3"
+tags:
+- blueocean
+author: michaelneale
+---
+
+As we get closer to
+link:/projects/blueocean[Blue Ocean]
+1.0, which is planned for the end of March, I have started
+link:/blog/2017/01/13/blueocean-dev-log-jan/[highlighting]
+some of the good stuff that has been going on, and this week was a _very_ busy week.
+
+A bunch of time was spent preparing for SCM API changes, which
```

Comment:
I'm not sure I'd mention this scm api roll-out-and-back as part of the good stuff.  Why mention this in the Blue Ocean dev log?